eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionReportWriter.java

```diff
@@ -84,31 +111,65 @@ public void setAccessionPrefix(String accessionPrefix) {
         this.accessionPrefix = accessionPrefix;
     }
 
+    /**
+     * We do not load the variants written in previous failed executions, but there might be duplicates hard to avoid,
+     * @see AccessionReportWriterTest#resumeWritingWithRepeatedVariant
+     */
     public void open(ExecutionContext executionContext) throws ItemStreamException {
         boolean isHeaderAlreadyWritten = IS_HEADER_WRITTEN_VALUE.equals(executionContext.get(IS_HEADER_WRITTEN_KEY));
-        if (output.exists() && !isHeaderAlreadyWritten) {
-            logger.warn("According to the job's execution context, the accession report should not exist, but it does" +
-                                " exist. The AccessionReportWriter will append to the file, but it's possible that " +
-                                "there will be 2 non-contiguous header sections in the report VCF. This can happen if" +
-                                " the job execution context was not properly retrieved from the job repository.");
-        }
+
         try {
-            boolean append = true;
-            this.fileWriter = new BufferedWriter(new FileWriter(this.output, append));
-            if (!isHeaderAlreadyWritten) {
-                writeHeader();
-                executionContext.put(IS_HEADER_WRITTEN_KEY, IS_HEADER_WRITTEN_VALUE);
+            if (isHeaderAlreadyWritten) {
+                // we are resuming a job
+                if (contigsOutput.exists() && variantsOutput.exists()) {
+                    loadContigMappingFromTemporaryFile(contigsOutput);
+                    boolean append = true;
```

Comment:
Since the append variable's value isn't being changed, should this be set here?

Reply:
another value is used later in this function, and I think 

```
this.contigsWriter = new BufferedWriter(new FileWriter(this.contigsOutput, true));
[...]
this.contigsWriter = new BufferedWriter(new FileWriter(this.contigsOutput, false));
```
is more obscure than
```
boolean append = true;
this.contigsWriter = new BufferedWriter(new FileWriter(this.contigsOutput, append));
[...]
boolean append = false;
this.contigsWriter = new BufferedWriter(new FileWriter(this.contigsOutput, append));
```